### ISSUE TITLE
LoeweLutzFabbriSeveri SAN model

### ISSUE THREAD

**Issue by hectormartineznavarro** [opencor/opencor#2720]
Hi,

I just wanted to let you know that the above-mentioned cell model contained in the default OpenCOR repository does not work.
Maybe I ve done smth wrong but I do not think so!

Best wishes and thanks!!

**Comment by agarny** [comment]
Hi, may I ask what you meann by the default OpenCOR repository? **This** is the default OpenCOR repository (on GitHub) and the models that get shipped with OpenCOR are the three models you can find [here](https://github.com/opencor/opencor/tree/master/models). As you can see, it doesn't include the LoeweLutzFabbriSeveri SAN model. So, would you mind telling us:
1. where you got that model from (URL?); and
2. what exactly you tried to run that model in OpenCOR (e.g., which solver settings did you use)?

In other words, please provide us with all the information needed to reproduce the issue you found.

**Comment by hectormartineznavarro** [comment]
Huh, I see.

Well, I downloaded the latest version of OpenCOR. Then I browsed through the little top left window (PMR) until I found that model, double-clicked and couldn’t be loaded, probably because some errors are found when parsing it.

I guess PMR accesses a repository containing cell models, and the version of that model in that repository does not work.

[signature_678981363]<https://www.ox.ac.uk/>

Dr Hector Martinez-Navarro
Senior Postdoctoral Researcher
Junior Research Fellow, Kellogg College
Computational Cardiovascular Science team
University of Oxford

**Comment by agarny** [comment]
Ok, thanks for the clarification.

FWIW, PMR stands for the [Physiome Model Repository](https://models.physiomeproject.org/welcome) and is repository that contains various CellML-encoded models (among other things). OpenCOR makes use of it through the `PMR` and the `PMR Workspaces` windows. The `PMR` window lists all the CellML-encoded models that are publically available.

Anyway, back to the [Loewe-Lutz-Fabbri-Severi model](https://models.physiomeproject.org/e/7cd), I can confirm that the model is broken. Among other things, it contains a MathML error:

![Screenshot 2023-02-07 at 11 02 15](https://user-images.githubusercontent.com/602265/217098037-93032854-9f28-494a-a5f5-d2041c72688a.png)

I am afraid that this has nothing to do with OpenCOR (hence closing this "issue") and the best way forward for you is to contact the authors about this problem.
